.travis.yml

```diff
@@ -3,8 +3,6 @@ install:
   - shards install
 script:
   - crystal spec
-  # Make sure everything works even without LuckyMigrator
-  - crystal spec -D without-migrator
```

Comment:
No point in having LuckyRecord and not LuckyMigrator. So I'm adding LuckyMigrator so specs and code are a bit simpler. At some later point Migrator and Record will be merged since they are also so intertwined. It's a pain to open separate PRs to make updates